go.mod

```diff
@@ -193,3 +193,5 @@ replace (
 	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.33.0
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.33.0
 )
+
```

Comment:
Update to upstream linodego when this PR is merged: https://github.com/linode/linodego/pull/791